Assets/Scripts/Enemies/StraightSpinner.cs

```diff
@@ -9,8 +9,8 @@ public override Vector3 PositionTransform(EnemyPosition position, float deltaTim
         return up * deltaTime;
     }
 
-    public override Vector3 RotationTransform(EnemyPosition position, float deltaTime)
+    public override Quaternion RotationTransform(EnemyPosition position, float deltaTime)
     {
-        return new Vector3(0, 0, RotationInDegrees);
+        return position.Rotation.Rotate(0, 0, RotationInDegrees);
     }
 }
```

Comment:
position.Rotation.Rotate will rotate the object won't it?  So why are you returning it as well?  Is that souly for testing?  If so I don't think we should do it like that.